specification/network/resource-manager/Microsoft.Network/stable/2020-07-01/virtualNetwork.json

```diff
@@ -1570,6 +1574,25 @@
             "modelAsString": true
           }
         },
+        "peeringSyncLevel": {
+          "type": "string",
+          "description": "The peering sync status of the virtual network peering.",
+          "enum": [
+            "FullySynced",
+            "RemoteUnsynced",
+            "LocalUnsynced",
+            "LocalAndRemoteUnsynced"
+          ],
+          "x-ms-enum": {
+            "name": "VirtualNetworkPeeringLevel",
+            "modelAsString": true
+          }
+        },
+        "syncRemoteAddressSpace":  {
+          "type": "boolean",
+          "default": false,
```

Comment:
`default: false` means SDK would provide default value if user doesn't input it. Is that what we want?

Reply:
Yes, the default value has to be false